web/src/app/users/UserNotificationRuleForm.tsx

```diff
@@ -1,11 +1,34 @@
 import React from 'react'
-import p from 'prop-types'
 import Grid from '@mui/material/Grid'
 import TextField from '@mui/material/TextField'
 import { FormContainer, FormField } from '../forms'
 import UserContactMethodSelect from './UserContactMethodSelect'
 
-export default function UserNotificationRuleForm(props) {
+interface UserNotificationRuleFormProps {
+  userID: string
+
+  value: Value
```

Comment:
We can use the `UserNotificationRule` type here from the `schema.d.ts` instead of defining a new type